File: python/ex097.py
# Refaça o exercício 91, só que agora em forma de função Maior(), mas faça uma
# adaptação que vai receber TRÊS números como parâmetro e vai retornar qual foi o
# maior entre eles.

def Maior(valueOne,valueTwo,valueThree):
    
    if(valueOne > valueTwo and valueOne > valueThree):
        return print(f"O maior valor é: {valueOne}")
    elif(valueTwo > valueOne and valueTwo > valueThree):
        return print(f"O maior valor é: {valueTwo}")
    elif(valueThree > valueOne and valueThree > valueTwo):
        return print(f"O maior valor é: {valueThree}")
    else:
        return print("Os três valores são iguais")

def Main():
    firstValue = int(input("Digite um valor: "))
    secondValue = int(input("Digite outro valor: "))
    thirdValue = int(input("Digite outro valor: "))
    
    return Maior(firstValue,secondValue,thirdValue)

Main()

File: typescript/src/ex097.ts
// Refaça o exercício 91, só que agora em forma de função Maior(), mas faça uma
// adaptação que vai receber TRÊS números como parâmetro e vai retornar qual foi o
// maior entre eles.

var readline = require('readline-sync');


function Major(valueOne: number, valueTwo: number, valueThree : number) {
    if (valueOne > valueTwo && valueOne > valueThree) {
        return valueOne;
    }else if(valueTwo > valueOne && valueTwo > valueThree){
        return valueTwo;
    }else{
        return valueThree;
    }
}

function MainMethodMaster(){
    let valueOne:number = parseInt(readline.question("Digite um valor: "));
    let valueTwo:number = parseInt(readline.question("Digite outro valor: "));
    let valueThree:number = parseInt(readline.question("Digite outro valor: "));

    console.log(Major(valueOne,valueTwo,valueThree));
}

MainMethodMaster();
